configs/centripetalnet/centripetalnet_hourglass104_mstest_16x6_210e_coco.py

```diff
@@ -103,3 +103,6 @@
     warmup_ratio=1.0 / 3,
     step=[190])
 runner = dict(type='EpochBasedRunner', max_epochs=210)
+
+# NOTE: This is for automatically scaling LR, USER CAN'T CHANGE THIS VALUE
+mmdet_official_special_batch_size = 96  # (16 GPUs) x (6 samples per GPU)
```

Comment:
We can both use `default_batch_size` for unity.

Reply:
OK, I will use `default_batch_size` in each config file instead of `mmdet_official_special_batch_size`

Reply:
Done